packages/truffle-contract/test/util.js

```diff
@@ -42,6 +42,12 @@ var util = {
     };
     const options = {
       contracts_directory: path.join(__dirname, "sources"),
+      logger: {
+      log(stringToLog) {
+        this.loggedStuff = this.loggedStuff + stringToLog;
+      },
+      loggedStuff: ""
+    },
```

Comment:
Why didn't prettier get the formatting here? Strange.

Anyway, do you think it's worth respecting the `quiet` option and including that as a convenience?

Reply:
Oh yeah! Forgot about that.